sdk/appconfiguration/Azure.Data.AppConfiguration/src/SettingSelector.cs

```diff
@@ -51,25 +51,7 @@ public class SettingSelector
         /// Creates a <see cref="SettingSelector"/> that will retrieve all <see cref="ConfigurationSetting"/> entities in the
         /// configuration store by setting both Key and Label filters to Any.
         /// </summary>
-        public SettingSelector() : this(Any, Any) { }
-
-        /// <summary>
-        /// Creates a <see cref="SettingSelector"/> that will retrieve <see cref="ConfigurationSetting"/> entities that match
-        /// the passed-in keys and labels.
-        /// </summary>
-        /// <param name="key">A key or key filter indicating which <see cref="ConfigurationSetting"/> entities to select.</param>
-        /// <param name="label">A label or label filter indicating which <see cref="ConfigurationSetting"/> entities to select.</param>
-        public SettingSelector(string key, string label = default)
-        {
-            Keys = new List<string>
-            {
-                key ?? Any
-            };
-
-            Labels = new List<string>();
-            if (label != null)
-                Labels.Add(label);
-        }
+        public SettingSelector() { }
```

Comment:
I thought we had decided to default this to Any, Any.  Is that not correct?  

Whichever it is, we should update the docstring to reflect it.

Reply:
There is no difference between `null`/`null` and `Any`/`Any`, since omitted value is equal to `*`: https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md#filtering

`null`/`null` pair seems more obvious.

Reply:
I don't have a super strong feeling on this, but in general I'm more in favor of being explicit about these things.  A '*' clearly indicates what's happening, whereas a user has to go to the service docs to learn what "no value" indicates.

Reply:
> A `*` clearly indicates what's happening

Only if user knows what `*` means, which also requires to go to service docs. Here it least user can read from the docstring of constructor itself that default object returns everything.

To me, it is both more "natural" to have type default with most of its fields set to type's defaults (which is why I also don't like that `SettingFields.All` is `uint.MaxValue` instead of `0`) and it matches the behavior of the REST API (where _no value_ means _filter not specified_ for any filter parameter).

Reply:
Sounds good.